pkg/api/common/versions.go

```diff
@@ -396,6 +397,7 @@ var AllKubernetesSupportedVersionsAzureStack = map[string]bool{
 	"1.23.15": true,
 	"1.24.7":  false,
 	"1.24.9":  true,
+	"1.25.7":  true,
 }
```

Comment:
Are we leaving 1.23.15 as supported for now? Plans to remove in the future?

Reply:
will do now

Reply:
Thanks, also need to change the defaults to 1.24 here then: https://github.com/Azure/aks-engine-azurestack/blob/master/pkg/api/common/const.go#L58